[PR] Fixing layout and item card on web devices

LuchoTurtle opened the pull request:
closes #6 

This removes the packages that were previously being used and uses a simple `LayoutBuilder` custom widget that conditionally renders according to screen width breakpoints.

Additionally, if the item card's timer is running and the person wants to complete the todo, that's now possible (without having to stop the timer).

LuchoTurtle committed:
fix: Responsive layout.

LuchoTurtle committed:
feat: Person can mark it as done while stopwatch is running.

LuchoTurtle committed:
fix: Fixing README.

nelsonic commented:
@LuchoTurtle _almost_ there ...

<img width="707" alt="image" src="https://github.com/dwyl/flutter-bloc-tutorial/assets/194400/9eb97ef1-219f-4c3c-8ee9-97ec149b2f2d">

Want to see why the coverage has decreased and then re-assign? 😉

LuchoTurtle committed:
feat: Adding tests that encompass new changes.

Test include actions on new screen sizes and tapping while the timer is ongoing.

LuchoTurtle commented:
@nelsonic Sorry about that. It should be back at 100%.

nelsonic reviewed (approved):
Thanks for fixing @LuchoTurtle 🤞🏼
